.env.sample

```diff
@@ -1,3 +0,0 @@
-# Copy this file to .env and put the right environment variables
-
-SLACK_API_TOKEN=
```

Comment:
the `.env` file is gitignored, the `.env.sample` is not gitignored, so when someone clones the repo can have a sample of the `.env` that they need, that's why it needs to stay in the repo.